dffml/df/base.py

```diff
@@ -193,6 +193,20 @@ def wrap(func):
         if not "conditions" in kwargs:
             kwargs["conditions"] = []
 
+        primitive_types = ["int", "float", "str", "bool"]
+
+        if not "inputs" in kwargs:
+            sig = str(inspect.signature(func))
+            sig = [
+                arg.strip()
+                for arg in sig.replace("(", "").replace("(", "").split(",")
+            ]
+
+            for arguements in sig:
+                arg, argtype = arguements.split(":")
+                if argtype in primitive_types:
+                    Definition(name=arg, primitive=str(argtype))
```

Comment:
We should make `kwargs["inputs"]` a empty dict.

Each key should be the name of the parameter, the value should be this newly created `Definition`